### ISSUE TITLE
"NaN" USD - Total Spend

### ISSUE THREAD

**Issue by stefanw1337** [dolkensp/HangarXPLOR#30]
So with Opera I get this error. It can't summarize anymore the total spent. <--That's also how you spell it.

So I figured out that it's because the UEC items is in there, as soon as they are filtered away, it all works again. So please fix this error. Seems to be only on my Opera client(not using Chrome extension for Opera) and for Chrome it's working as expected.

**Comment by dolkensp** [comment]
Yeah, I need to publish the latest updates to opera. I'll add it to the top of my to-do list.

**Comment by q220** [comment]
Having the same issue with Firefox

**Comment by justnice980** [comment]
I started having this issue on chrome 78.0.3904.108

**Comment by paulpv** [comment]
This was supposedly fixed w/ #45 
I am seeing it too and turning on debug logging the problem looks like the following stray item at the end of the list of ships:
```
<li>
    <h4 class="empy-list">Your hangar is empty.</h4>
</li>
```

**Comment by paulpv** [comment]
Ding ding; it happens when there are exactly 10 items on the last page and the 11th page results in an empty hangar.
The problem is a typo.
RSI calls the empty list `empy-list`.
@dolkensp calls it `empty-list`

https://github.com/dolkensp/HangarXPLOR/blob/develop/src/web_resources/HangarXPLOR.Render.js#L38
```
buffer.push($('<h4>', { class: 'empty-list', text: 'Your search returned no results.' }));
```

**Comment by paulpv** [comment]
May have spoken too soon.
That may not be the precise problem, but it is in this area.

**Comment by paulpv** [comment]
Not exactly related, but close.
The problem is in the call to `$items.each(HangarXPLOR.ProcessItem);` when there are no items on the page:
https://github.com/dolkensp/HangarXPLOR/blob/develop/src/web_resources/HangarXPLOR.ProcessPage.js#L11
I confirmed that adding a "isEmpty" check before calling that fixes the NaN.

**Comment by dolkensp** [comment]
Dammit, thanks @paulpv - I needed to isEmpty a little sooner! I'll fix that asap

@stefanw1337 - Spent is past-tense only, Spend is present/future tense, and here I use it in present tense to indicate that you're able to adjust that spend by melting and rebuying ships up until the point at which CIG locks things down after go-live ;)

https://www.difference.wiki/spend-vs-spent/

**Comment by Petosiris** [comment]
> Yeah, I need to publish the latest updates to opera. I'll add it to the top of my to-do list.

Also, the latest release doesn't have a link to Opera and Readme misses links to both Opera and Edge.
